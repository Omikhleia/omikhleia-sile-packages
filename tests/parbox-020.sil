\begin{document}
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 020 = From the package documentation, alignment with character struts

(2A) \parbox[width=37%fw, valign=top, strut=character, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\smallskip

(2B) \parbox[width=37%fw, valign=bottom, strut=character, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\smallskip

(2C) \parbox[width=37%fw, valign=middle, strut=character, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\end{document}