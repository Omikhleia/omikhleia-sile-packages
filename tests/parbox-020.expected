Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	50.2488
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	32.9327
T	21 36 w=10.6445	(2A)
Mx 	43.5773
T	12 w=3.1689	())
Draw line	49.4238	42.0945	198.2268	0.5000
Draw line	49.4238	88.6902	198.2268	0.5000
Draw line	49.4238	42.0945	0.5000	47.0957
Draw line	247.6506	42.0945	0.5000	47.0957
Mx 	49.4238
T	82 81 72 w=15.1660	(one)
My 	62.2488
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	78.2510
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.9278
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	134.3331
T	86 76 87 w=10.0146	(sit)
Mx 	148.8243
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.9845
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.6213
T	86 68 w=8.4521	(sa)
Mx 	231.0734
T	71 76 83 w=13.2080	(dip)
Mx 	244.2814
T	16 w=3.3691	(-)
Mx 	49.4238
My 	74.2488
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	73.4215
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	49.4238
My 	86.2488
T	87 90 82 w=15.5322	(two)
Mx 	29.7638
My 	136.8445
T	11 w=3.1689	(()
Mx 	32.9327
T	21 37 w=10.1904	(2B)
Mx 	43.1232
T	12 w=3.1689	())
Draw line	48.9697	92.6902	198.2268	0.5000
Draw line	48.9697	139.2859	198.2268	0.5000
Draw line	48.9697	92.6902	0.5000	47.0957
Draw line	247.1965	92.6902	0.5000	47.0957
Mx 	48.9697
My 	100.8445
T	82 81 72 w=15.1660	(one)
My 	112.8445
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	77.7969
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.4737
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	133.8790
T	86 76 87 w=10.0146	(sit)
Mx 	148.3703
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.5305
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.1672
T	86 68 w=8.4521	(sa)
Mx 	230.6194
T	71 76 83 w=13.2080	(dip)
Mx 	243.8274
T	16 w=3.3691	(-)
Mx 	48.9697
My 	124.8445
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	72.9675
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	48.9697
My 	136.8445
T	87 90 82 w=15.5322	(two)
Mx 	29.7638
My 	169.4402
T	11 w=3.1689	(()
Mx 	32.9327
T	21 38 w=10.0537	(2C)
Mx 	42.9864
T	12 w=3.1689	())
Draw line	48.8330	143.2859	198.2268	0.5000
Draw line	48.8330	189.8816	198.2268	0.5000
Draw line	48.8330	143.2859	0.5000	47.0957
Draw line	247.0598	143.2859	0.5000	47.0957
Mx 	48.8330
My 	151.4402
T	82 81 72 w=15.1660	(one)
My 	163.4402
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	77.6602
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.3370
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	133.7423
T	86 76 87 w=10.0146	(sit)
Mx 	148.2336
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.3937
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.0305
T	86 68 w=8.4521	(sa)
Mx 	230.4826
T	71 76 83 w=13.2080	(dip)
Mx 	243.6907
T	16 w=3.3691	(-)
Mx 	48.8330
My 	175.4402
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	72.8308
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	48.8330
My 	187.4402
T	87 90 82 w=15.5322	(two)
End page
Finish
